File: dataset/output-bk/MISTRAL_codestral-latest/Task123_MISTRAL_codestral-latest.java
// Java
public class Session {
    private int userId;
    private boolean isActive;

    public Session(int userId) {
        this.userId = userId;
        this.isActive = true;
    }

    public void endSession() {
        this.isActive = false;
    }

    // Usage
    public static void main(String[] args) {
        Session userSession = new Session(123);
        System.out.println(userSession.isActive);  // true
        userSession.endSession();
        System.out.println(userSession.isActive);  // false
    }
}

File: dataset/output/MISTRAL_codestral-latest/Task123_MISTRAL_codestral-latest.c
// C
#include <stdio.h>
#include <stdbool.h>

typedef struct {
    int userId;
    bool isActive;
} Session;

Session createSession(int userId) {
    Session session;
    session.userId = userId;
    session.isActive = true;
    return session;
}

void endSession(Session* session) {
    session->isActive = false;
}

// Usage
int main() {
    Session userSession = createSession(123);
    printf("%d
", userSession.isActive);  // 1 (true)
    endSession(&userSession);
    printf("%d
", userSession.isActive);  // 0 (false)
    return 0;
}